Update readmes to match common design (Search category) 1/1 (#488)
Esri/arcgis-runtime-samples-java: search/find-address/README.md

# Find address

Find the location for an address.

![Image of find address](FindAddress.gif)

## Use case

A user can input a raw address into your app's search bar and zoom to the address location.

## How to use the sample

> Select an entry or type in an address in from the drop-down menu at the top of the screen. Suggestions will appear as text is entered. Click a suggestion or hit return to see the address marked with a pin. Tapping on the pin will show the address in a callout.

Reviewer: ```suggestion
Select an entry from the drop-down menu at the top left of the screen to zoom to the address, marked with a pin. 
```
